index.js

```diff
@@ -78,7 +78,7 @@ var validateMessage = function(raw) {
     }
 
     if (TYPES !== '*' && TYPES.indexOf(type) === -1) {
-      error('"%s" is not allowed type !', type);
+      error('"%s" is not allowed type ! Valid types are: %s', type, config.types.join(', '));
```

Comment:
I did that to expose the default types on the instance, for testability. The tests for the module depend implicitly on the custom types declared in `package.json`, and so i thought that might one day change and make tests mysteriously fail. This guards against that.